bosshud/ze_l0v0l_v1_4_csgo2.txt

```diff
@@ -39,35 +39,40 @@
 	{
 		"Type"				"breakable"
 		"BreakableName"		"bosslvl4_minigun_hit"
+		"CustomText"		"Minigun"
 	}	
 
 	"6"
 	{
 		"Type"				"breakable"
 		"BreakableName"		"bosslvl4_launcher_hit"
+		"CustomText"		"Giant Launcher"
 	}	
 
 	"7"
 	{
 		"Type"				"breakable"
 		"BreakableName"		"bosslvl4_beam_hit"
+		"CustomText"		"Beam"
 	}	
 
 	"8"
 	{
 		"Type"				"breakable"
 		"BreakableName"		"bosslvl4_tank_hit"
+		"CustomText"		"Tank"
 	}
 
 	"9"
 	{
 		"Type"				"breakable"
 		"BreakableName"		"bosslvl4_hit"
+		"CustomText"		"Queen"

```

Comment:
As I know you're already aware, l0v0l uses vscript boss HP. None of this will work.